101-vm-with-data-management-gateway/README.md

```diff
@@ -0,0 +1,34 @@
+# Create a data management gateway and make it workable in azure VM
+
+Before deploying the template you must have the following
+
+1. **Data Factory.** The gateway is created in the data factory. If you don't have a data factory,  see the [Create data factory](https://docs.microsoft.com/en-us/azure/data-factory/data-factory-move-data-between-onprem-and-cloud#create-data-factory) for steps to create one.
```

Comment:
yes, ADF team has a sample. this link contains five options, Azure portal, Visual studio, powershell, ARM template and REST API